app/controllers/order_management/order_details_controller.rb

```diff
@@ -101,8 +101,8 @@ def load_order_statuses
     return if @order_detail.reconciled?
 
     if @order_detail.complete?
-      @order_statuses = [OrderStatus.complete, OrderStatus.canceled, OrderStatus.unrecoverable]
-      @order_statuses << OrderStatus.reconciled if @order_detail.can_reconcile?
+      @order_statuses = OrderStatus.where(name: %w[complete canceled unrecoverable reconciled]).to_a
```

Comment:
Note that this relies on mysql's case insensitive string comparison. I think we should set the status names to constants in the model and use a scope or something.

If you feel this is getting complex let's create another PR to tackle this